docs/self-managed/setup/deploy/local/c8run.md

```diff
@@ -44,6 +44,8 @@ When successful, a new Operate window automatically opens.
 
 :::note
 If Camunda 8 Run fails to start, run the [shutdown script](#shut-down-camunda-8-run) to end the current processes, then run the start script again.
+
+For Mac: The binary has not been signed yet and users may encounter the warning `"c8run" Not Openend`. You can follow the instructions of Apples support guide to [grant an exception](https://support.apple.com/en-us/102445)
```

Comment:
```suggestion
Mac users may encounter the warning `"c8run" Not Opened`. Follow the Apple support instructions to [grant an exception](https://support.apple.com/en-us/102445).
```

Does this capture the warning correctly? I am not sure we need to include whether or not the binary is signed.

Reply:
<img width="259" alt="image" src="https://github.com/user-attachments/assets/de870442-bfa3-4bdf-885e-10b55aef464a" />

I think your changes are sufficient, and we can iterate on it if necessary.

Reply:
👍 Updated to this version!

Reply:
To your point @conceptualshark, this is the first time I've noticed "Running software that hasn’t been signed and _notarized_". I've always referred to this as signed software. All the more reason to not include this language and stick. to what the user is seeing and how they can accomplish their intended goal.